service/access.go

```diff
@@ -248,7 +250,16 @@ func (a *accessService) getAccessToken(ctx context.Context, domain, role, proxyF
 	if !ok {
```

Comment:
on line 249, `tok, ok := ...` => `cd, ok := ...`
change variable name as the data type changed.

Reply:
corrected in [41ab134](https://github.com/AthenZ/athenz-client-sidecar/pull/13/commits/41ab134d10c10301774379986172948b96e824de)